src/main/java/io/buildpacks/example/sample/CalculatorService.java

```diff
@@ -23,4 +23,35 @@ public int divide(int a, int b) throws IllegalArgumentException {
         }
```

Comment:
1. Consider providing more specific exception types instead of `IllegalArgumentException` for the `divide` method. This can help in distinguishing different types of exceptions more easily.